denizen_scripts/hub1/repo-link/Adriftus_Moderator_Panel/panels/mod_ban_inv.dsc

```diff
@@ -0,0 +1,51 @@
+# -- BAN PANEL
+mod_ban_inv:
+  type: inventory
+  debug: false
+  title: <&6>A<&e>MP <&f>› <&c>Ban <&6><player.flag[amp_map].as_map.get[uuid].as_player.name> from <player.flag[amp_map].as_map.get[from].to_titlecase>.
+  inventory: CHEST
+  size: 54
+  definitions:
+    x: <item[air]>
+    b1: <item[yellow_stained_glass_pane].with[display_name=<&r>]>
+    b2: <item[orange_stained_glass_pane].with[display_name=<&r>]>
+    back: <item[red_stained_glass_pane].with[display_name=<&c><&l>↩<&sp>Actions<&sp>panel;nbt=<list[to/actions]>]>
+    head: <item[mod_player_item]>
+  procedural items:
+    - define inventory:<list>
```

Comment:
for `set` data actions using `define`, the syntax is preferred `- define <name> <object>`
Additionally, you're defining a list here, the proper syntax for inserting a list into a definition would be `:->` for single/empty entries, `:|:` for multiple entries